### PR TITLE
Even more ambience

### PR DESCRIPTION
**Pull request by moonheart08**
## About the PR 

**Screenshots**

**Changelog**

:cl:
- add: Nanotrasen finally figured out that still water in hydroponics trays attracts space mosquitos and installed micropumps.
- add: Nanotrasen has finally installed ventilation on their R&D servers.
- add: The volume and quality of the game's ambience can be adjusted in settings.

### CODE REVIEW

File: Resources/Audio/Ambience/Objects/license.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
 circular_saw.ogg - https://freesound.org/people/derjuli/sounds/448133/ and clipped - CC0-1.0
 gas_pump - https://freesound.org/people/karinalarasart/sounds/441419/ - CC0-1.0
-gas_hiss - https://freesound.org/people/geodylabs/sounds/122803/ - CC BY 3.0
+gas_hiss - https://freesound.org/people/geodylabs/sounds/122803/ - CC-BY-3.0
 gas_vent - https://freesound.org/people/kyles/sounds/453642/ - CC0-1.0
+flowing_water_open - https://freesound.org/people/sterferny/sounds/382322/ - CC0-1.0
+server_fans - https://freesound.org/people/DeVern/sounds/610761/ - CC -BY-3.0
```
Comment (ike709) [inline Resources/Audio/Ambience/Objects/license.txt:6]:
```suggestion
server_fans - https://freesound.org/people/DeVern/sounds/610761/ - CC-BY-3.0
```

File: Content.Client/Audio/AmbientSoundSystem.cs [hunk 2]
```diff
@@ -33,25 +34,28 @@ public sealed class AmbientSoundSystem : SharedAmbientSoundSystem
         private float _maxAmbientRange;
         private float _cooldown;
         private float _accumulator;
+        private float _ambienceVolume = 0.0f;
 
         /// <summary>
         /// How many times we can be playing 1 particular sound at once.
         /// </summary>
-        private int _maxSingleSound = 8;
+        private int MaxSingleSound => (int) (_maxAmbientCount / (16.0f / 6.0f));
```
Comment (moonheart08) [inline Content.Client/Audio/AmbientSoundSystem.cs:42]: At the default setting, this means 6 simultaneous sounds, at the max, 24.

File: Content.Shared/CCVar/CCVars.cs [hunk 3]
```diff
@@ -22,23 +22,40 @@ public sealed class CCVars : CVars
...
+        /// The minimum value the user can set for ambience.max_sounds
+        /// </summary>
+        public static readonly CVarDef<int> MinMaxAmbientSourcesConfigured =
+            CVarDef.Create("ambience.min_max_sounds_configured", 16, CVar.REPLICATED | CVar.SERVER);
+
+        /// <summary>
+        /// The maximum value the user can set for ambience.max_sounds
+        /// </summary>
+        public static readonly CVarDef<int> MaxMaxAmbientSourcesConfigured =
+            CVarDef.Create("ambience.max_max_sounds_configured", 64, CVar.REPLICATED | CVar.SERVER);
```
Comment (metalgearsloth) [inline Content.Shared/CCVar/CCVars.cs:52]: This should be a cheat value if it's supposed to be a hardcap.

**Review by mirrorcult — APPROVED** [review]

File: Content.Shared/CCVar/CCVars.cs (hunk 3 above)
Comment (moonheart08) [reply Content.Shared/CCVar/CCVars.cs:52]: Cheat value value?

**Pull request merged**